python/ray/tests/test_metrics.py

```diff
@@ -236,7 +236,7 @@ def cpu_resources(actor_info):
         if child_actor_info["state"] == -1:
             assert child_actor_info["requiredResources"]["CustomResource"] == 1
         else:
-            assert child_actor_info["state"] == 1
+            assert child_actor_info["state"] == 2
```

Comment:
Why is it changed?

Reply:
> Why is it changed?

That's because that the new ActorState is added at the very beginning of the Enum. The state we expected here is ActorState::ALIVE, so the value should be changed to 2.